test/e2e/e2e_test.go

```diff
@@ -87,6 +102,88 @@ func pollForServiceServingSecret(client *kubernetes.Clientset, secretName, names
 	})
 }
 
+func pollForCABundleInjectionConfigMap(client *kubernetes.Clientset, configMapName, namespace string) error {
+	return wait.PollImmediate(time.Second, 10*time.Second, func() (bool, error) {
+		_, err := client.CoreV1().ConfigMaps(namespace).Get(configMapName, metav1.GetOptions{})
+		if err != nil && errors.IsNotFound(err) {
+			return false, nil
+		}
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	})
+}
+
+func editServiceServingSecretData(client *kubernetes.Clientset, secretName, namespace string) error {
```

Comment:
This should be addToServiceServingSecretData() and we can make editServiceServingSecretData() a function for another test where we fiddle with the existing data and ensure that the old secret persists.

Reply:
changed it